frontend/www/js/omegaup/components/course/StudentProgress.test.ts

```diff
@@ -0,0 +1,46 @@
+import { shallowMount } from '@vue/test-utils';
+import expect from 'expect';
+import Vue from 'vue';
+import T from '../../lang';
+import { omegaup } from '../../omegaup';
+
+import course_StudentProgress from './StudentProgress.vue';
+
+describe('StudentProgress.vue', () => {
+  it('Should handle scores', async () => {
+    const expectedDate = new Date('1/1/2020, 12:00:00 AM');
+    const wrapper = shallowMount(course_StudentProgress, {
+      propsData: {
+        course: {
+          alias: 'hello',
+        },
+        assignment: [
```

Comment:
would it be possible to put the type annotation to all entries directly under `propsData`?

```suggestion
        assignment: <omegaup.Assignment[]>[
```

the types should match the ones declared in `StudentProgress.vue`. same for `course` and `student`.